libs/messaging/db-message-bus-impl/src/main/kotlin/net/corda/messagebus/db/persistence/DbUtils.kt

```diff
@@ -21,7 +22,8 @@ internal fun EntityManagerFactoryFactory.create(
     val dbSource = if (jdbcUrl == null || jdbcUrl.contains("hsqldb", ignoreCase = true)) {
         InMemoryDataSourceFactory().create(DB_MESSAGE_BUS)
     } else {
-        PostgresDataSourceFactory().create(
+        DBBaseDataSourceFactory(HikariDataSourceFactory()).create(
```

Comment:
```suggestion
        DBBaseDataSourceFactory().create(
```
No point adding the default arg here.